docs/source/tutorials/traits_ui_scientific_app.rst

```diff
@@ -375,25 +309,11 @@ Qt toolkit by default to create its widget. They can be turned live
 and displayed by starting a Qt application, and its main loop (ie event
 loop in Qt speech).
 
+    .. literalinclude:: code_snippets/event_loop_qt.py
+        :language: python
 
-    `code snippet #3 <../_static/event_loop_qt.py>`_
-
-    .. code-block:: python
-
-        from pyface.qt.QtGui import QApplication
-
-        from traits.api import HasTraits, Int
-
-
-        class Counter(HasTraits):
-            value = Int()
-
-
-        Counter().edit_traits()
-        QApplication.instance().exec_()
-
-There is a `similar example for wxPython <../_static/event_loop_wx.py>`_
-and a `toolkit-independent example that uses Pyface <../_static/event_loop.py>`_
+There is a `similar example for wxPython <code_snippets/event_loop_wx.py>`_
+and a `toolkit-independent example that uses Pyface <code_snippets/event_loop.py>`_
```

Comment:
these two hyper links are broken at the moment.

Reply:
i'm not sure how but the two files are still included in "_static" so for now we can probably revert this change.